includes/blocks/course-theme/class-course-theme.php

```diff
@@ -63,6 +64,7 @@ protected function should_initialize_blocks() {
 	public function initialize_blocks() {
 		new Course_Title();
 		new Site_Logo();
+		new Focus_Mode();
```

Comment:
Will this also be used in post_types other than lesson? I am asking because I see the `<!-- wp:sensei-lms/focus-mode-toggle /-->` only i `single-lesson.php`. If we only expect to use this in the lessons maybe it can be initialized only for lessons?

Again it is probable that I missed something 😅

Reply:
Yeah I started adding more global things there, we'll see what's needed where. But tbh that lesson/quiz separation is not really needed since just registering these blocks won't have any side-effect when they are not used. Also in some cases quiz pages will be using the lesson template, so the `if`s here will probably be removed.

Reply:
Got it! Thanks for the explanation :)